services/121-service/src/payments/reconciliation/safaricom-reconciliation/safaricom-reconciliation.controller.ts

```diff
@@ -35,6 +35,11 @@ export class SafaricomReconciliationController {
   public async processTransferCallback(
     @Body() safaricomTransferCallback: SafaricomTransferCallbackDto,
   ): Promise<void> {
+    console.log(
+      'POST fsps/safaricom/transfer-callback:',
+      safaricomTransferCallback,
+    );
+
```

Comment:
is this left in intentionally? If the reason is that the timeout callback below also has it, then I can explain that. We were never able to test that timeout callback on sandbox, so that's why we put in the console.log, so that once it would first come in on production we would know. But for this transfer callback we do now, and it's not needed, and it will just flood our logs. So it can be removed.

Reply:
I would remove it again (and put comment on architecture chit chat about potentially more logging in 121-service)